sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Primitives/TaskExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static void WaitAndUnwrapException(this Task task)
         /// </summary>
         /// <param name="task">The task. May not be <c>null</c>.</param>
         /// <param name="cancellationToken">A cancellation token to observe while waiting for the task to complete.</param>
-        /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was cancelled before the <paramref name="task"/> completed, or the <paramref name="task"/> raised an <see cref="OperationCanceledException"/>.</exception>
+        /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was canceled before the <paramref name="task"/> completed, or the <paramref name="task"/> raised an <see cref="OperationCanceledException"/>.</exception>
```

Comment:
Is this a mistake as well?

Reply:
Nah, that's just a comment.   The others were tokens that have to be in sync with the server values.